src/common/hotbar-store.ts

```diff
@@ -94,7 +94,7 @@ export class HotbarStore extends BaseStore<HotbarStoreModel> {
   }
 
   @action
-  protected async fromStore(data: Partial<HotbarStoreModel> = {}) {
+  protected fromStore(data: Partial<HotbarStoreModel> = {}) {
```

Comment:
These cannot be `async`. See https://github.com/lensapp/lens/pull/3178

Reply:
Are we sure this change does not have any side effects?

Reply:
This actually fixed the fromStore not working